cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/RatioTapChangerConversion.java

```diff
@@ -87,8 +87,11 @@ public void convert() {
         }
 
         rtca.setLoadTapChangingCapabilities(ltcFlag);
```

Comment:
Is it something that should remain in `RatioTapChangerConversion`?

Reply:
I will depend on the final behaviour of the `loadTapchangingCapabilities` attribute. If finally `ltcFlag` and `tapChangerControl` are not related it is ok.